.github/workflows/bot.yml

```diff
@@ -21,6 +21,16 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    # Prevent multiple simultaneous runs per repository target (issue or PR)
+    # Different event types store the target number in different locations:
+    # - issues/issue_comment: github.event.issue.number
+    # - pull_request: github.event.pull_request.number  
+    # - pull_request_review: github.event.review.pull_request.number
+    # - pull_request_review_comment: github.event.comment.pull_request.number
+    concurrency:
+      group: bot-${{ github.repository }}-${{ github.event.issue.number || github.event.pull_request.number || github.event.review.pull_request.number || github.event.comment.pull_request.number || 'unknown' }}
+      cancel-in-progress: true
```

Comment:
This seems like a good general solution covering a wider range of scenarios than the original bug mentioned. Let's keep this, but also remove the workflow trigger on review comments. Testing suggests that creating a single comment also creates a review, so we can just trigger on the overall review and not on individual comments